topbeat/system/process.go

```diff
@@ -24,6 +25,12 @@ type Process struct {
 	Ctime    time.Time
 }
 
+type ProcStats struct {
+	ProcStats bool
```

Comment:
@andrewkroh There are several variables that now should be renamed with the restructuring. To keep the changes to a minimum I didn't rename any variables. If it is ok for you I would prefer to do this in a second PR.